ee.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
```

Comment:
Where should we place this file?

Reply:
Could eventually be part of the docs as a sort of demo evaluator?

Reply:
Is it useful to have it around for you? If not, should we leave it in a separate branch for the moment?